examples/forms-signup.rs

```diff
@@ -313,14 +377,37 @@ struct FakeApiRequest {
 
 #[derive(Debug)]
 enum FakeApiResponse {
-    SignUpFailure(Map<SignupField, String>),
+    // the API returns numbers, which needs to be mapped to a specific error message
+    SignUpFailure(Vec<u32>),
```

Comment:
Why would a Rust API use u32's instead of an enum to represent the error? I think by introducing this extra step this is complicating the example more than it needs to.